.github/CODEOWNERS

```diff
@@ -1,235 +1,223 @@
-/src/sinks/gcp @bruceg
-/src/sinks/http.rs @jszwedko
-/src/sinks/humio_logs.rs @jszwedko
-/src/sinks/splunk_hec.rs @ktff
-/src/sinks/kafka.rs @lukesteensen
-/src/sinks/prometheus/collector.rs @bruceg
-/src/sinks/prometheus/remote_write.rs @bruceg
-/src/sinks/prometheus/exporter.rs @lukesteensen
-/src/sinks/splunk_hec.rs @lukesteensen
-/src/sinks/statsd.rs @lukesteensen
-/src/sinks/tcp.rs @bruceg
-/src/sinks/tcp.rs @lukesteensen
-/src/sinks/util/adaptive_concurrency @bruceg
-
-/src/sources/apache_metrics/ @jszwedko
-/src/sources/aws_kinesis_firehose/ @jszwedko
-/src/sources/aws_s3.rs @jszwedko
-/src/sources/docker.rs @fanatid
-/src/sources/generator.rs @bruceg
-/src/sources/host_metrics.rs @bruceg
-/src/sources/internal_logs.rs @bruceg
-/src/sources/journald.rs @bruceg
-/src/sources/mongodb_metrics/ @fanatid
-/src/sources/nginx/ @fanatid
-/src/sources/postgresql_metrics.rs @fanatid @jszwedko
-/src/sources/prometheus @bruceg
-/src/sources/stdin.rs @bruceg
-/src/sources/syslog.rs @lukesteensen
-/src/sources/tcp.rs @bruceg
-/src/sources/udp.rs @bruceg
-/src/sources/vector.rs @lukesteensen
-/src/sources/util/multiline_config.rs @MOZGIII @JeanMertz
-/src/sources/kubernetes_logs/ @timberio/vector-kubernetes
-/src/sources/splunk_hec.rs @ktff
-
-/src/topology/ @lukesteensen
-
-/src/transforms/add_tags.rs @lukesteensen
-/src/transforms/ansi_stripper.rs @JeanMertz
-/src/transforms/aws_cloudwatch_logs_subscription_parser.rs @jszwedko
-/src/transforms/coercer.rs @bruceg
-/src/transforms/filter.rs @bruceg
-/src/transforms/field_filter.rs @lukesteensen
-/src/transforms/geoip.rs @lukesteensen
-/src/transforms/grok_parser.rs @lukesteensen
-/src/transforms/key_value_parser.rs @Nick011 @bruceg
-/src/transforms/log_to_metric.rs @lukesteensen
-/src/transforms/merge.rs @MOZGIII
-/src/transforms/regex_parser.rs @lukesteensen
-/src/transforms/remap.rs @JeanMertz @FungusHumungus
-/src/transforms/remove_tags.rs @lukesteensen
-/src/transforms/sample.rs @lukesteensen
-/src/transforms/split.rs @lukesteensen
-/src/transforms/tokenzier.rs @lukesteensen
-
-/src/wasm/ @hoverbear @JeanMertz
-
-/tests/buffering.rs @MOZGIII
-/tests/data/wasm/ @hoverbear @JeanMertz
-/tests/shutdown.rs @ktff
-/tests/cli.rs @ktff
-/tests/helm-snapshots/ @timberio/vector-kubernetes
-
-/.github/semantic* @binarylogic @jamtur01
-/.github/SUPPORT.md @binarylogic @jamtur01
-/.github/PULL_REQUEST_TEMPLATE.md @binarylogic @jamtur01
-/.github/ISSUE_TEMPLATE/ @binarylogic @jamtur01
-/.github/workflows/k8s_e2e.yml @timberio/vector-kubernetes
-/.github/workflows/ @jamtur01
+/src/mapping/ @timberio/vector-processing
+/src/metrics.rs @timberio/vector-core @timberio/vector-processing
+/src/prometheus.rs @timberio/vector-integrations
+/src/rusoto/ @timberio/vector-aws
+/src/sinks/ @timberio/vector-integrations
+/src/sinks/**/aws* @timberio/vector-aws
+/src/sinks/**/datadog* @timberio/vector-core
+/src/sinks/**/gcp* @timberio/vector-gcp
+/src/sinks/**/http* @timberio/vector-core @timberio/vector-integrations
```

Comment:
Do these (and others) really need the `**` wildcard? All of the AWS sinks match `/src/sinks/aws*`. All of the GCP sinks are in `/src/sinks/gcp/`. The last one matches both `/src/sinks/http.rs` which looks like it could be just @timberio/vector-integrations and `/src/sinks/util/http.rs` which is already covered below.

Reply:
They don't, I was just trying to defend against things being moved around, but we can just update this file.